stencil-library/src/components/table/table.tsx

```diff
@@ -0,0 +1,138 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { PagingInfo, pagingDefaults } from '../table/table-utils';
+
+interface ExtendedPagingInfo extends PagingInfo {
+  onPrev: () => void;
+  onNext: () => void;
+}
+
+const ExtendedPagingDefaults = {
```

Comment:
Should this be `const ExtendedPagingDefaults: ExtendedPagingInfo =`?

Reply:
Yes! Good catch.